contracts/CurrencyNetwork.sol

```diff
@@ -1199,6 +1230,23 @@ contract CurrencyNetwork is CurrencyNetworkInterface, Ownable, Authorizable, Des
         );
     }
 
+    function _increaseDebt(address debtor, address creditor, int72 value) internal {
```

Comment:
if you can actually reduce debt with it, maybe better call it addDebt

Reply:
I call it `addToDebt`